cypress/integration/preferences/public_key_management.ts

```diff
@@ -46,7 +46,7 @@ describe("Public Key Management Page", () => {
     it("Should include the public in the public key list after adding", () => {
       cy.dataCy("key-value-input").clear();
       cy.dataCy("key-value-input").type(pubKey, { delay: 0 });
-      cy.dataCy("save-key-button").click();
+      cy.contains("Save").click();
```

Comment:
is it possible to be more specific by calling `cy.contains("button", "Save")` or does that not work with LeafyGreen? (this comment applies across several files)

Reply:
Wow great find! I didn't know this was an option!